client/components/telecom/telephony/sidebar/telephony-sidebar.service.js

```diff
@@ -9,91 +9,114 @@ angular.module("managerApp").service("TelephonySidebar", function ($q, $translat
      */
     self.initTelephonySubsection = function () {
 
-        // fetch all telephony services and groups
-        return TelephonyMediator.getAll().then(function (groups) {
+        return telecomVoip.fetchAll().then(function (billingAccounts) {
 
-            // GROUPS
-            var sortedGroups = (_.values(groups) || []).sort(function (first, second) {
+            /* ----------  billingAccount display  ---------- */
+
+            // first sort by getDisplayedName
+            let sortedBillingAccounts = billingAccounts.sort(function (first, second) {
                 return first.getDisplayedName().localeCompare(second.getDisplayedName());
             });
-            _.forEach(sortedGroups, function (group) {
 
-                var subSection = SidebarMenu.addMenuItem({
-                    id: group.billingAccount,
-                    title: group.getDisplayedName(),
+            // add billingAccount subsections to telephony sidebar menu item
+            sortedBillingAccounts.forEach(function (billingAccount) {
```

Comment:
suggestion : maybe you could factorize forEach function in each of sorted items no ?

Reply:
ho yeah. I though to that and then I forgot :smile: